.github/CODEOWNERS

```
@@ -13,6 +13,11 @@
 /airbyte-integrations/bases/source-acceptance-tests/ @airbytehq/connector-extensibility
 /airbyte-integrations/connector-templates/ @airbytehq/connector-extensibility
 
+# Oauth
+/airbyte-oauth/ @airbytehq/connector-operations
+/airbyte-server/src/main/java/io/airbyte/server/handlers/OAuthHandler.java @airbytehq/connector-operations
+airbyte-server/src/test/java/io/airbyte/server/handlers/OAuthHandlerTest.java @airbytehq/connector-operations
```

nit, I think this is equivalent

```suggestion
/airbyte-server/src/test/java/io/airbyte/server/handlers/OAuthHandlerTest.java @airbytehq/connector-operations
```